job-search-react/src/App.test.js

```diff
@@ -6,3 +6,8 @@ test('renders learn react link', () => {
   const linkElement = screen.getByText(/learn react/i);
   expect(linkElement).toBeInTheDocument();
 });
+
+it('renders without crashing', () => {
```

Comment:
im guessing this test fails

Reply:
This is the only error I have left 

FAIL  src/App.test.js (28.195 s)
  ✕ renders learn react link (317 ms)

  ● renders learn react link

    TestingLibraryElementError: Unable to find an element with the text: /learn react/i. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.